lib/database/models/promo.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:objectbox/objectbox.dart';
+
+import 'coupon.dart' show Coupon;
+import 'resto.dart' show Resto;
+
+@Entity()
+class Promo {
+  @Id()
+  int id = 0;
+
+  @Backlink('promo')
+  final coupons = ToMany<Coupon>();
+
+  final resto = ToOne<Resto>();
+
+  int amountOfCoupons = 0;
+  int timeToComplete = 0; // in seconds
+
+  @Property(type: PropertyType.date)
+  DateTime startDate = DateTime.now();
+
+  @Property(type: PropertyType.date)
+  DateTime? endDate; // TODO: maybe named it as expiredDate or exirationDate
+
+  String description = '';
```

Comment:
NIT Is it needed to set a default value?

Reply:
what does NIT mean ?

Reply:
NIT is the short of Not Important Thing.